libevm/VM.cpp

```diff
@@ -15,12 +15,34 @@
     along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include "interpreter.h"
 #include "VM.h"
 
 using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
+evm_instance* interpreter_create()
```

Comment:
Should this be in `extern "C"` if the header says so?

Reply:
Seems to be working on my machine :D

Reply:
I think the header may force it into that.